lib/kindle_highlights/client.rb

```diff
@@ -4,7 +4,6 @@ class CaptchaError < StandardError; end
     class AuthenticationError < StandardError; end
     class AsinNotFoundError < StandardError; end
 
-    KINDLE_LOGIN_PAGE      = "https://read.amazon.com/notebook"
```

Comment:
We can keep this and do something like:

```ruby
KINDLE_ROOT = "https://read.amazon.com"
KINDLE_LOGIN_PAGE = "#{KINDLE_ROOT}/notebook"
```

Reply:
Simply doing so doesn't work since ruby interpret class variables on `require`.
To delay the evaluation, I think we need something like:
```ruby
def kindle_login_page:
   @kindle_login_page || "#{KINDLE_ROOT}/notebook"
end
```

Maybe you'll prefer the idea that give up monkeypatching like:
```ruby
def initialize(email_address:, password:, mechanize_options: {}, kindle_root: 'https://read.amazon.com')
  @kindle_login_page =  "#{KINDLE_ROOT}/notebook"
  #...
end
```